src/main/java/seedu/address/logic/parser/FilterCommandParser.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.logic.commands.FilterCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new FindCommand object
+ */
+public class FilterCommandParser implements Parser<FilterCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindCommand
+     * and returns a FindCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public FilterCommand parse(String args) throws ParseException {
+        String trimmedArgs = args.trim();
+        if (trimmedArgs.isEmpty()) {
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, FilterCommand.MESSAGE_USAGE));
+        }
+
+        return new FilterCommand(trimmedArgs);
```

Comment:
Just a question, if a user were to input "filter  positive" instead of "filter positive". The difference being a double white space inbetween, will the command be parsed correctly?

Reply:
It will still be parsed correctly, because the .trim() function used in the FilterCommandParser class removes all of the trailing and leading whitespaces of the input.

Reply:
Noted! Good job with the method 👍